modules/log6x-collection-setup.adoc

```diff
@@ -102,7 +102,7 @@ Annotations
 <7> logs: Refers to the log resources that can be created.
 <8> verbs: The actions allowed on the resources.
 <9> create: Grants permission to create new logs in the Loki system.
-----
+
```

Comment:
> === Writing audit logs

[All titles and section headings must have an anchor ID](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-titles-and-section-headings). This and the other headings within this module do not have an ID.